change/@fluentui-react-next-2020-04-22-17-09-48-functional-checkbox.json

```diff
@@ -1,8 +0,0 @@
-{
-  "type": "prerelease",
```

Comment:
I think you might have had a bad merge or pull, I don't think this should be deleted.